docs/defi/fcm/index.md

```diff
@@ -0,0 +1,184 @@
+    FYV[FYV<br/>Flow Yield<br/>Vaults] --> FCM
+    MOET[MOET<br/>Synthetic<br/>Stablecoin] --> FCM
+
+    style FCM fill:#f9f,stroke:#333,stroke-width:4px
+    style ALP fill:#bbf,stroke:#333,stroke-width:2px
+    style FYV fill:#bfb,stroke:#333,stroke-width:2px
+    style MOET fill:#fbb,stroke:#333,stroke-width:2px
+```
+
+### The Three Components
+
+1. **[ALP (Automated Lending Platform)](../alp/index.md)**: The core lending/borrowing engine
+   - Manages collateral deposits and debt positions
+   - Provides automated rebalancing to maintain position health
+   - Uses DeFi Actions for composability
+   - Implements liquidation prevention mechanisms
+
+2. **[FYV (Flow Yield Vaults)](#)**: The yield aggregation layer
+   - Deploys borrowed capital into optimal yield strategies
+   - Automatically compounds returns
+   - Provides liquidity for ALP liquidation prevention
+   - Manages risk through auto-balancing
+
+3. **[MOET](#)**: The synthetic stablecoin
+   - Serves as the unit of account for all pricing
+   - Primary borrowed asset in ALP
+   - Medium of exchange between components
+   - Maintains stability through over-collateralization
+
+## How the Components Work Together
+
+FCM creates a **yield-generating flywheel** by connecting these three components:
+
+### The Capital Flow
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant ALP
+    participant MOET
+    participant FYV
+    participant Yield
+
+    User->>ALP: 1. Deposit FLOW collateral
+    ALP->>ALP: 2. Calculate borrowing capacity
+    ALP->>MOET: 3. Auto-borrow MOET
+    MOET->>FYV: 4. Deploy to yield strategy
+    FYV->>Yield: 5. Generate returns
+    Yield->>FYV: 6. Accumulate yield
+
+    Note over ALP,FYV: When collateral price drops:
+    FYV->>ALP: 7. Provide funds for rebalancing
+    ALP->>MOET: 8. Repay debt automatically
+
+    Note over User,ALP: Result: Position stays healthy
+```
+
+### Step-by-Step Flow
+
+1. **User deposits collateral** (e.g., FLOW tokens) into an ALP position
+2. **ALP auto-borrows** MOET against the collateral to reach target health ratio (1.3)
+3. **Borrowed MOET flows** to a FYV strategy (via DrawDownSink)
+4. **FYV deploys capital** into yield-generating opportunities (farms, LPs, etc.)
+5. **Yield accumulates** and compounds automatically
+6. **If collateral price drops**: FYV provides liquidity to ALP (via TopUpSource)
+7. **ALP repays debt** automatically to prevent liquidation
+8. **User keeps position healthy** without manual intervention
+
+## Key Innovations
+
+### 1. Yield-Powered Liquidation Prevention
+
+Unlike traditional lending protocols where you must manually add collateral or repay debt when prices drop, FCM **uses your yield to maintain position health**. Yield from FYV strategies flows back to ALP automatically, ALP pulls from FYV to repay debt when needed, your position stays healthy without manual intervention, and **you earn yield while protecting yourself from liquidation**.
```

Comment:
'.... ALP automatically via scheduled transactions (link to doc)' - useful explaining how it works incase they want to dig deeper